### ISSUE TITLE
Verify quibble works with esm

### ISSUE THREAD

**Issue by searls** [testdouble/quibble#13]
So, @jdalton released [esm](https://github.com/standard-things/esm) today and it looks super cool!

@schoonology just raised the point that we should validate that quibble works with esm and figure out why not if it doesn't. There's a chance things will "just work" since we're monkey patching Module._load, but we should test it out!

**Comment by novemberborn** [comment]
One problem is that esm exports live bindings through getters. That seems to prevent proper fakes from being returned.

**Comment by searls** [comment]
Could you please create a minimal project that reproduces the issue?

**Comment by pingortle** [comment]
👋 I've been playing with the combination of esm and testdouble.js on a small project recently, ran into some issues with that combination and wanted to share my findings here.

The bad news is that quibble does not Just Work™ out of the box with esm enabled. The good news is that I think I've found a fairly simple workaround.

Here is an [example repository](https://github.com/pingortle/quibble-plus-esm) where I've reproduced the issue.

**TL;DR** add this before your test suite runs to get esm + quibble working together.
```js
require('quibble').ignoreCallsFromThisFile(require.resolve('esm/esm'))
```

I can submit a PR to update the README with a note about esm support so it's easily discoverable. I would also be interested in adding esm support based on this approach as a baked in feature for quibble.

cc @jasonkarns

**Comment by searls** [comment]
Fixed by #29
